src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/ConstantNameCheck.java

```diff
@@ -61,10 +61,23 @@
  * <p>
  * To configure the check:
  * </p>
+ * <p>
+ * The following configuration is applicable by default.
+ * </p>
```

Comment:
Please remove this section, there is already a section above

Reply:
@Vyom-Yadav ^^^

Reply:
done.